protocol/rv.go

```diff
@@ -119,7 +119,7 @@ type RvDirective struct {
 	// Other
 
 	Delay      time.Duration
-	External   bool
+	External   *[]byte // always non-nil when instruction sent (even with no value)
```

Comment:
From the spec:

> This contains a CBOR array with at least one element. The first element of the array is a string that specifies the external mechanism to use. The subsequent array elements are specific to this mechanism.
>
> There are no RVExtRv mechanisms defined in this specification.

So perhaps it makes sense to parse the "mechanism" and "args" into two fields.